site/content/docs/5.1/examples/dropdowns/index.html

```diff
@@ -107,19 +107,19 @@
     </form>
     <ul class="list-unstyled mb-0">
       <li><a class="dropdown-item d-flex align-items-center gap-2 py-2" href="#">
-        <span class="d-inline-block bg-success rounded-circle" style="width: .5em; height: .5em;"></span>
+        <span class="badge rounded-pill bg-success d-inline p-1"></span>
```

Comment:
Simply adding `.p-1` and removing `style` attribute ensures the same result. If this relates to #35480, then I'd recommend using the utility-based pattern instead of requiring a component change.

Reply:
@ffoodd I'm don't change component